### PR TITLE
Update troubleshoot postgres error case - Closes #21

### LINKED ISSUE #21: Troubleshooting: 2 binary nodes on 1 server
Add error case, if 2 Binary nodes run on the same server (e.g. 1 testnet, 1 mainnet)

Related issue: LiskHQ/lisk-scripts/issues/187

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #21

### CODE REVIEW

File: troubleshooting/troubleshooting.md [hunk 1]
```diff
@@ -14,23 +14,38 @@
...
+PostgreSQL is already installed on your system and listening to the postgreSQl default port 5432.
+This can happen e.g. when a second Lisk Core node is installed on the same server.
+
+#### Solution 1:
+Change the port of your already installed postgreSQL instance to an unused and available port number.
+Locate the corresponding configuration file for PostgreSQL, `postgresql.conf`, and change the port to e.g. `5433`.
+```bash
+locate postgresql.conf 
+```
+After chaning the config, restart the corresponding postgreSQL process.
```
Comment (Nazgolze) [inline troubleshooting/troubleshooting.md]: changing

File: troubleshooting/troubleshooting.md [hunk 2]
```diff
@@ -14,23 +14,38 @@
...
+
+#### Solution 2:
+The error can be ignored by setting the ignore flag `-i`.
+In that case, the installation script `installLisk.sh` will be executed regardless of the above error.
+The script might run successfully to the end, but ignoring the error might result in unwanted interferences between the different postgreSQL instances.
+```bash
+bash installLisk.sh install -r main -i
+```
+
+#### Solution 3:
```
Comment (Nazgolze) [inline troubleshooting/troubleshooting.md:46]: I might also suggest shutting down and disabling the global install as an option in addition to removing it if there's data that the user doesn't want to lose

`systemctl stop postgresql`
`systemctl disable postgresql`

**Review by Nazgolze — CHANGES_REQUESTED** [review]

File: troubleshooting/troubleshooting.md (hunk 2 above)
Comment (Nazgolze) [reply troubleshooting/troubleshooting.md:46]: (I forget if purging will delete the data. I'll test this out quick - I happen to have a system where I can do this :grin:)

Comment (Nazgolze) [reply troubleshooting/troubleshooting.md:46]: Yeah, that nuked my data

Comment (Tschakki) [reply troubleshooting/troubleshooting.md:46]: thanks for testing :)

File: troubleshooting/troubleshooting.md [hunk 3]
```diff
@@ -14,23 +14,45 @@
...
+If postgreSQL has been installed globally on the system, disable it temporary:
```
Comment (Nazgolze) [inline troubleshooting/troubleshooting.md]:
```suggestion
If postgreSQL has been installed globally on the system, disable it:
```

Comment (Nazgolze) [reply troubleshooting/troubleshooting.md]: or `temporarily`

File: troubleshooting/troubleshooting.md [hunk 4]
```diff
@@ -14,23 +14,45 @@
...
+```bash
+systemctl stop postgresql
```
Comment (Nazgolze) [inline troubleshooting/troubleshooting.md]:
```suggestion
sudo systemctl stop postgresql
```

File: troubleshooting/troubleshooting.md [hunk 5]
```diff
@@ -14,23 +14,45 @@
...
+systemctl disable postgresql
```
Comment (Nazgolze) [inline troubleshooting/troubleshooting.md]:
```suggestion
sudo systemctl disable postgresql
```

Comment (Nazgolze) [reply troubleshooting/troubleshooting.md]: my bad, I forgot to mention sudo before

**Review by Nazgolze — APPROVED** [review]

**Pull request merged**